### DISCUSSION TITLE
Sending logs to kafaka

### DISCUSSION THREAD

**Discussion by epifanoviv** [Security-Onion-Solutions/securityonion#1794] [Unsupported Versions] [answered]
Hello ! Please tell me where should i configure security-onion to send logs to kafka?
Thank's a lot!

**Comment by weslambert** [comment, marked as answer]
We don't support this at the moment but may consider supporting the output plugin in the future:
https://www.elastic.co/guide/en/logstash/current/plugins-outputs-kafka.html
